builtin/providers/google/resource_google_service_account_test.go

```diff
@@ -9,6 +9,14 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+var (
+	projectId = multiEnvSearch([]string{
+		"GOOGLE_PROJECT",
+		"GCLOUD_PROJECT",
+		"CLOUDSDK_CORE_PROJECT",
+	})
+)
```

Comment:
This is a simpler version of a `schema.MultiEnvDefaultFunc`. Could abstrace that out in `google/provider.go` and re-use here. Will add a TODO for that.